API.md

```diff
@@ -1764,12 +1764,23 @@ Requires the string value to be a valid hexadecimal string.
 const schema = Joi.string().hex();
 ```
 
-#### `string.base64()`
+#### `string.base64([padding])`
```

Comment:
I'm not sure what the consensus is but I'd personally prefer to see this as an options object instead that way we can support other options in the future without making a breaking change.